bot/exts/utilities/bookmark.py

```diff
@@ -72,6 +72,10 @@ class Bookmark(commands.Cog):
     def __init__(self, bot: Bot):
         self.bot = bot
 
+    async def cog_load(self) -> None:
```

Comment:
For reviewers:
Should we maybe move this to `bot.__main__`?

Reply:
I think it makes the most sense to only sync the tree either A) on command or B) in `setup_hook` on bot startup

Reply:
That's what I had in mind, in a "general" setup hook for the bot, not per Cog, but I thought `__main__` would be that.
But I just browser and I saw that `Bot` has a setup hook as well, I'll move it there

Reply:
this likely makes more sense in bot core

Reply:
That's true, I'll move this to a different PR

Reply:
https://github.com/python-discord/bot-core/pull/171